### PR TITLE
Further text refactoring.

### PR DESCRIPTION
**Pull request by Maascha**
Relates to
https://github.com/corona-school/project-user/issues/522

### CODE REVIEW

File: src/pages/SingleCourse.tsx [hunk 1]
```diff
@@ -516,7 +518,7 @@ function PupilJoinCourseAction({ subcourse, refresh }: { subcourse: Pick<Subcour
             >
                 <Modal.Content>
                     <Modal.CloseButton />
-                    <Modal.Header>Kurseinformationen</Modal.Header>
+                    <Modal.Header></Modal.Header>
```
Comment (Jonasdoubleyou) [inline src/pages/SingleCourse.tsx:521]: Empty header?

File: src/pages/SingleCourse.tsx [hunk 2]
```diff
@@ -699,7 +701,7 @@ function PupilLeaveWaitingListAction({ subcourse, refresh }: { subcourse: Pick<S
             >
                 <Modal.Content>
                     <Modal.CloseButton />
-                    <Modal.Header>Kursinformationen</Modal.Header>
+                    <Modal.Header></Modal.Header>
```
Comment (Jonasdoubleyou) [inline src/pages/SingleCourse.tsx:704]: Here too?

File: src/pages/SingleCourse.tsx [hunk 3]
```diff
@@ -761,7 +763,7 @@ function PupilJoinWaitingListAction({ subcourse, refresh }: { subcourse: Pick<Su
             <Modal isOpen={isOnWaitingListModal} onClose={() => setOnWaitingListModal(false)}>
                 <Modal.Content>
                     <Modal.CloseButton />
-                    <Modal.Header>Kursinformationen</Modal.Header>
+                    <Modal.Header></Modal.Header>
```
Comment (Jonasdoubleyou) [inline src/pages/SingleCourse.tsx:766]: deliberate?

**Review by Jonasdoubleyou — APPROVED** [review]

File: src/pages/SingleCourse.tsx (hunk 3 above)
Comment (Maascha) [reply src/pages/SingleCourse.tsx:772]: As discussed,yes

**Pull request merged**